openc3-init/plugins/packages/openc3-tool-common/src/tools/base/AppNav.vue

```diff
@@ -177,6 +177,18 @@ export default {
       }
     )
   },
+  methods: {
+    newTabUrl(tool) {
+      let url = null
+      if (tool.url[0] == '/' && tool.url[1] != '/') {
```

Comment:
How would the url[1] character ever be '/'?

Reply:
http/https agnostic urls.
//openc3.com